src/test/java/stepDefinitions/MyStepDefinitions.java

```diff
@@ -0,0 +1,58 @@
+package stepDefinitions;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import pages_sample.*;
+
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class MyStepDefinitions {
```

Comment:
Please make a meaningful name, in this case, ShopMainPageSteps is more suitable.